posthog/hogql_queries/insights/trends/test/test_trends_query_runner.py

```diff
@@ -3844,7 +3844,7 @@ def test_trends_event_multiple_breakdowns_combined_types(self):
             Breakdown(type="group", group_type_index=0, property="industry"),
         ]
 
-        for breakdown_filter in itertools.permutations(breakdowns, 2):
+        for breakdown_filter in itertools.combinations(breakdowns, 3):
```

Comment:
should we do combinations 2 instead of combinations 3?

this will halve it again, correct? from 56 to 28.

Reply:
Initially, I thought keeping three combinations would be safer for a regression test. I couldn’t find a good reason to do so now, so I changed it to two.